docs/04-database-communication.md

```diff
@@ -130,3 +130,17 @@ Sometimes more advanced tasks need to be performed on the database. For those oc
 ```dart
 var result = await session.db.query('SELECT * FROM mytable WHERE ...');
 ```
+
+### View
+View will consider as Table in Serverpod. You can use the same way to query data from view. But you can't insert, update or delete data from view. For now, you need to manually create your view in database and then declare in protocol how we declare the table.
+
+Example you have Company table that have OwnerId as FK on Person Table, now you need one view to show the company name and person name. You can create view in protocol like this:
+```yaml
+class: CompanyView
+table: company_view
+asView: true
```

Comment:
I think a better syntax would be to either allow `table:` or `view:` as that would align more with the Postgres syntax (`create table ...` vs `create view ...`. There should be a check so that we do not both provide a table and a view.

```yaml
class: CompanyView
view: company_view
fields:
  name: String
  ownerName: String

Reply:
I thought of it but in the end I put asView so I  no need to modify a lot of file, but after you mention it, yes I'm agree for the cleanest of the code better to define view or table only on yaml. 
Just to align with you so the output of the generated code will be like 
```dart
class CompanyView extends ViewRow {
  @override
  String get className => 'CompanyView';
  @override
  String get viewName => 'company_owner_view';
 .
 .
 .
}

class CompanyViewView extends View {
  @override
  String get viewName => 'company_owner_view';
}
```
Thanks appreciate your feedback..